### PR TITLE
CSS bug fixes

### LINKED ISSUE #27: Home Page: Div Centering on Large Screens + Reorder Img Div
image

### LINKED ISSUE #31: Sponsorship Page: Text Alignment with Sponsors
image

### PR DESCRIPTION
**Pull request by annsudhart**
- (Sponsor page) Sponsor logo images now align with header (resolves #31)
- (Home page) ACM UCSD logo now appears on right side of home page (resolves #27)
- (Home page) Centered div content above the fold for wider screens and iPad screen sizes (resolves #27)
- (Events page) Aligned event header text with event cards

The other PR (#61) I made was from a fork I made, so I switched to a PR from a branch I made

### CODE REVIEW

**Comment by StormFireFox1** [comment]
You should close the other PR once this one is merged.

File: src/pages/about-us/index.tsx [hunk 1]
```diff
@@ -9,18 +9,21 @@ const AboutUs: React.FC = () => (
...
-          <b>cybersecurity</b>
-          <span>, and </span>
-          <b>graphics</b>
-          <span> to name a few. We welcome you to drop by our events! </span>
+            <b>design</b>
+            {', '}
+            <b>hacking</b>
+            {', '}
+            <b>cybersecurity</b>
+            {', '}
```
Comment (annsudhart) [inline src/pages/about-us/index.tsx:20]: Yes I know this looks horrible, but this is the only way I got linting to work

File: src/pages/events/style.less [hunk 2]
```diff
@@ -65,7 +65,7 @@
   .events-header {
     flex-direction: column;
     align-items: flex-start;
-    padding: 1em;
+    padding: 1em 2.5em;
     h1 {
```
Comment (steets250) [inline src/pages/events/style.less:69]: The padding here doesn't line up with the event cards.

**Review by steets250 — APPROVED** [review]
Looks good, all 4 fixes mentioned were present.

**Review by leestanley — APPROVED** [review]

**Pull request merged**